demo/gtm/phoebe_demo_server_container.json

```diff
@@ -1336,7 +1341,7 @@
antity\": 2,\n        \"index\": 1\n      },\n      {\n        \"item_id\": \"blazer\",\n        \"item_name\": \"Blazer\",\n        \"price\": 100,\n        \"quantity\": 1,\n        \"index\": 2\n      }],\n      \"parameters\": {}\n    });\n\n    runCode(mockVariableData).then((resp) => {\n      assertThat(resp).isString();\n      assertThat(resp).isEqualTo(\"180\");\n    });\n- name: Check default returned on error status code\n  code: |\n    const mockVariableData = {\n      projectNumber: \"11111111111\",\n      vertexEndpointID: \"1234567891011121314\",\n      cloudLocation: \"europe-west2\",\n      addRequestDataToItems: true,\n      defaultValueOnError: \"0\"\n    };\n\n    generateMockItems([\n      {\"item_id\": \"shoes\", \"item_name\": \"Shoes\", \"revenue\": 80, \"quantity\": 2},\n    ]);\n\n    generateMockVertexAI([], 500);\n\n    expectedPayload = JSON.stringify({\n      \"instances\":[{\n        \"item_id\": \"shoes\",\n        \"item_name\": \"Shoes\",\n        \"price\": 80,\n        \"quantity\": 2,\n        \"index\": 1\n      }],\n      \"parameters\": {}\n    });\n\n    runCode(mockVariableData).then((resp) => {\n      assertThat(resp).isString();\n      assertThat(resp).isEqualTo(\"0\");\n    });\nsetup: |-\n  const JSON = require(\"JSON\");\n  const Promise = require(\"Promise\");\n\n  const purchasedProducts = [];\n  let vertexAIResonse;\n  let expectedPayload;\n\n  /**\n   * Build the mock items from the event.\n   * This method changes the global purchasedProducts variable, which is used in\n   * the mock logic.\n   * @param {!Array<number>} items - the items to mock.\n   */\n  function generateMockItems(items) {\n    for (const item of items) {\n      purchasedProducts.push({\n        \"item_id\": item.item_id,\n        \"item_name\": item.item_name,\n        \"price\": item.revenue,\n        \"quantity\": item.quantity\n      });\n    }\n  }\n\n  /**\n   * Build the mock response from VertexAI.\n   * This method changes the global vertexAIResonse variable, which is used in\n   * the mock logic.\n   * @param {!Array<number>} predictions - the predictions to use in the response.\n   * @param {number} statusCode - the status code to use in the VertexAI response.\n   */\n  function generateMockVertexAI(predictions, statusCode) {\n    const predictionString = predictions.join(', ');\n    vertexAIResonse = {\n      \"statusCode\": statusCode,\n      \"body\":\"{\\\"predictions\\\": [\" + predictionString + \"]}\"\n    };\n  }\n\n  // Change sendHttpRequest to return our mocked VertexAI response.\n  mock(\"sendHttpRequest\", (url, requestOptions, postBody) => {\n    assertThat(postBody).isEqualTo(expectedPayload);\n    return Promise.create((resolve) => {\n      resolve(vertexAIResonse);\n    });\n  });\n\n  // Inject our products into the event data.\n  mock(\"getEventData\", (data) => {\n    if (data === \"items\") {\n      return purchasedProducts;\n    }\n  });\n\n\n___NOTES___\n\nCreated on 5/3/2023, 5:16:28 PM\n\n\n"
```

Comment:
What was the change you made here?

By the looks of things you're adding the new variable template to this import?

I'd be cautious about doing this, as it will just mean that there are two places to update the code, so this copy will likely become stale (as it's not obvious).

This demo was originally used in a workshop, so I don't think we need to be too active here.

Unless you have a good reason that I'm missing?